build.gradle

```diff
@@ -33,24 +33,23 @@ configurations {
 // The repackaging rules are defined in the "shadowJar" task below.
 dependencies {
     compile "com.android.tools:r8:1.0.37"
-    compile "com.android.tools.build:apkzlib:3.2.0-alpha06"
-    compile "com.android.tools.ddms:ddmlib:26.2.0-alpha04"
-
+    compile "com.android.tools.build:apkzlib:3.2.0"
+    compile "com.android.tools.ddms:ddmlib:26.2.0"
     shadow "com.android.tools.build:aapt2-proto:0.3.1"
     shadow "com.google.auto.value:auto-value:1.5.2"
-    apt "com.google.auto.value:auto-value:1.5.2"
+    annotationProcessor "com.google.auto.value:auto-value:1.5.2"
     shadow "com.google.errorprone:error_prone_annotations:2.2.0"
     shadow "com.google.guava:guava:26.0-jre"
-    shadow "com.google.protobuf:protobuf-java:3.4.0"
-    shadow "com.google.protobuf:protobuf-java-util:3.4.0"
+    shadow "com.google.protobuf:protobuf-java:3.6.1"
```

Comment:
If you could switch to 3.6.0 instead, that would be helpful. We can't use 3.6.1 on our internal branch at the moment.